### DISCUSSION TITLE
deploying-asp-net-core-applications-to-kubernetes-part-3-deploying-applications-with-helm/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#114] [Announcements]
# deploying-asp-net-core-applications-to-kubernetes-part-3-deploying-applications-with-helm/

In this post I provide an overview to Helm, and discuss some of the benefits it can provide for managing and deploying your applications.

https://andrewlock.net/deploying-asp-net-core-applications-to-kubernetes-part-3-deploying-applications-with-helm/

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at October 04 2020, 01:42, in reply to [Kaushik Roy Chowdhury](https://disqus.com/by/kaushik_roy_chowdhury/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/114#discussioncomment-6059422)</em>

---

<p>Hi Kaushik,</p><p>Of course, please do 🙂 What's your software channel?</p><p>Cheers<br>Andrew</p>

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at October 23 2020, 08:53, in reply to [Kaushik Roy Chowdhury](https://disqus.com/by/kaushik_roy_chowdhury/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/114#discussioncomment-6059425)</em>

---

<p>Really sorry to hear that Kaushik 🙁 My thoughts are with you, take care</p>

[a comment by an automated account is not shown]

**Reply by akhanalcs** [reply]
Bro, Kubernetes is a Container Orchestrator. If you don't have containers, what are you orchestrating?

**Comment by akhanalcs** [comment]
Your posts are amazing!
